src/lib/components/Modal/Modal.jsx

```diff
@@ -243,52 +142,43 @@ export class Modal extends React.Component {
 }
 
 Modal.defaultProps = {
-  actions: [],
   autoFocus: true,
   children: null,
+  closeButtonRef: null,
   id: undefined,
-  onClose: null,
   portalId: null,
   position: 'center',
-  scrollView: (<ScrollView
-    customEndShadowStyle={{ background: 'radial-gradient(farthest-side at center bottom, rgba(0, 0, 0, 0.16) 0%, rgba(0, 0, 0, 0.06) 40%, rgba(0, 0, 0, 0.02) 85%, rgba(0, 0, 0, 0) 100%)' }}
-    customStartShadowStyle={{ background: 'radial-gradient(farthest-side at center top, rgba(0, 0, 0, 0.15) 0%, rgba(0, 0, 0, 0.05) 60%, rgba(0, 0, 0, 0.02) 85%, rgba(0, 0, 0, 0) 100%)' }}
-    shadowSize="16px"
-  />),
+  primaryButtonRef: null,
   size: 'medium',
 };
 
 Modal.propTypes = {
   /**
-   * Actions to be rendered in modal footer.
-   */
-  actions: PropTypes.arrayOf(PropTypes.shape({
-    color: PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'help', 'info', 'note', 'light', 'dark']),
-    disabled: PropTypes.bool,
-    feedbackIcon: PropTypes.node,
-    id: PropTypes.string,
-    label: PropTypes.string.isRequired,
-    onClick: PropTypes.func.isRequired,
-  })),
-  /**
-   * If `true`, focus the first action in the footer when the modal is opened.
+   * If `true`, focus the first input element in the modal or primary button (referenced by the `primaryButtonRef` prop)
+   * when the modal is opened.
    */
   autoFocus: PropTypes.bool,
   /**
-   * Content of the modal.
+   * Nested elements. Supported types are:
+   *
+   * * `ModalHead`
+   * * `ModalBody`
+   * * `ModalFooter`
+   *
+   * At least `ModalBody` is required.
    */
   children: PropTypes.node,
   /**
-   * ID of the root HTML element. It also serves as a base for nested elements:
-   * * `<ID>__content`
-   * * `<ID>__closeModalHeaderButton`
-   * * `<ID>__closeModalFooterButton`
+   * Reference to close button element. It is used to close modal when Escape key is pressed.
```

Comment:
It should say `…used to close modal when Escape key is pressed or the backdrop is clicked`